src/Features/Core/Portable/QuickInfo/IndentationHelper.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.QuickInfo
+{
+    // Reproduces logic in IProjectionBufferFactoryServiceExtensions (editor layer) 
```

Comment:
Can we sit the other guy on top of this helper and then translate the results of this to elision ranges?

I'm loathe to have hte same logic duplicated in two different places.